packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -376,31 +378,76 @@ export const components: ThemeOptions["components"] = {
   MuiCheckbox: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <CheckIcon />,
+      indeterminateIcon: <SubtractIcon />,
     },
     styleOverrides: {
       root: ({ theme }) => ({
-        borderRadius: "4px",
-        "&:hover": {
+        width: theme.spacing(4),
+        height: theme.spacing(4),
+        padding: 0,
+        borderRadius: theme.spacing(1),
+        borderWidth: "1px",
+        borderStyle: "solid",
+        borderColor: theme.palette.grey[500],
+
+        ".MuiSvgIcon-root": {
+          color: theme.palette.common.white,
+          width: theme.spacing(3),
+          height: theme.spacing(3),
+        },
+
+        "&.Mui-checked": {
+          backgroundColor: theme.palette.primary.main,
+          borderColor: theme.palette.primary.main,
+
+          ".MuiFormControlLabel-root:hover > &": {
+            backgroundColor: theme.palette.primary.dark,
+            borderColor: theme.palette.primary.dark,
+          },
+        },
+
+        ".MuiFormControlLabel-root:hover > &": {
           backgroundColor: "transparent",
+          borderColor: theme.palette.grey[900],
         },
-        padding: 0,
-        ".Mui-error > &": {
-          color: theme.palette.error.main,
-          "&:hover": {
-            color: theme.palette.error.dark,
+        ".Mui-error:hover > &": {
+          borderColor: theme.palette.error.dark,
+
+          "&.Mui-checked": {
+            backgroundColor: theme.palette.error.dark,
+            borderColor: theme.palette.error.dark,
           },
         },
-        ".Mui-error > &.Mui-checked": {
-          "&:hover": {
-            color: theme.palette.error.dark,
+        ".Mui-error > &": {
+          borderColor: theme.palette.error.main,
+
+          "&.Mui-checked": {
+            backgroundColor: theme.palette.error.main,
+            borderColor: theme.palette.error.main,
           },
         },
         "&.Mui-focusVisible": {
+          borderColor: theme.palette.grey[900],
           outlineColor: theme.palette.primary.main,
-          outlineOffset: 0,
+          outlineOffset: "2px",
           outlineStyle: "solid",
           outlineWidth: "2px",
         },
+        "&.Mui-disabled": {
+          backgroundColor: theme.palette.grey[50],
+          borderColor: theme.palette.grey[300],
+
+          ".Mui-error > &": {
+            backgroundColor: theme.palette.grey[50],
+            borderColor: theme.palette.grey[300],
+          },
+
+          ".MuiSvgIcon-root": {
+            color: theme.palette.common.black,
+          },
+        },
```

Comment:
@edburyenegren-okta Is there a way to import these classnames from somewhere, so we aren't tied to strings?

Reply:
This one is `.${svgIconClasses.root}`. I don't think there are variables available for the "global" classes tho.

For all of the components, you can grab them via `import { componentClasses } from "@mui/material/Component";` -- for ex: `import { listItemTextClasses } from "@mui/material/ListItemText";`